core/modules/xrdoss/QservOss.h

```diff
@@ -33,10 +33,10 @@
 // System headers
 #include <deque>
 #include <set>
+#include <memory>
```

Comment:
I guess most people do not really care about order or order them by criteria other than alphabetic.

The coding standard item 4-10 (https://confluence.lsstcorp.org/pages/viewpage.action?pageId=16908674#C++Files-IncludeFilesandIncludeStatements) tells that we should order includes (in one group) by dependency first and alphabetically after that. I'm not sure how to guess the dependencies between headers, so we should just probably use alphabetic ordering.